### PR TITLE
Switch to default tf.keras.layers.BatchNormalization

### PR DESCRIPTION
**Pull request by Maxinho96**
As stated in the official tf documentation, now setting trainable=False will run the BatchNormalization layer in inference mode, so when the layer is frozen it will use moving mean and moving variance instead of current batch mean and variance:
https://www.tensorflow.org/api_docs/python/tf/keras/layers/BatchNormalization

So, there is no need to use a custom BatchNormalization layer anymore.

### CODE REVIEW

**Comment by zzh8829** [comment]
thats a great news
